### PR TITLE
Path transitivity and its test

### PR DESCRIPTION
**Pull request by annacanal**
#139

### CODE REVIEW

File: braph/graph/measures/PathTransitivity.m [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            measure_class = 'PathTransitivity';
+        end
+        function name = getName()
+            name = 'Path-Transitivity';
+        end
+        function description = getDescription()
+            description = [ ...
+                'The path transitivity of a graph is the density of ' ...
+                'triangles to that are available along the ' ...
+                'shortest-paths between all pairs of nodes.' ...
```
Comment (egolol) [inline braph/graph/measures/PathTransitivity.m:103]: I dont understand this.

Comment (annacanal) [reply braph/graph/measures/PathTransitivity.m:103]: the **to** after triangles shouldn't be there.
so the density of triangles that are available along the shortest-paths between all pairs of nodes

**Comment by egolol** [comment]
@annacanal Anna, nice job, could you comment this measure? Thank you.

**Comment by annacanal** [comment]
I will comment this measure!
@mitemijalkov I am not sure about the plan, but maybe it is better to comment on all the measures before implementing more?

**Comment by egolol** [comment]
@annacanal , we have made some changes to all measures, please see Degree for an example. Main changes are within the constructor and a new function named getAvailableSettings(). @maverickyuwei  also important for your measure.

**Comment by annacanal** [comment]
Updated with the new Measure class!
I detected that in Degree and the rest of measures the new function is not added at the initial class documentation of the functions

**Comment by egolol** [comment]
yeah, theres an issue already for that.. will do that later in the week.

**Pull request merged**